src/components/Send/utils/createLinkUtils.ts

```
@@ -0,0 +1,208 @@
+        // step 4: store link in redux and confirm the claim link
+        dispatch(sendFlowActions.setLink(link))
+        console.log(`Submitting claim link confirm at ${new Date().getTime() - now}ms`)
+        await submitClaimLinkConfirm({
+            chainId: selectedChainID,
+            link,
+            password: password ?? '',
+            txHash: hash,
+            senderAddress: address ?? '',
+            amountUsd: parseFloat(usdValue ?? '0'),
+            transaction:
+                transactionType === 'not-gasless' ? preparedDepositTxs && preparedDepositTxs.unsignedTxs[0] : undefined,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)